src/backend/distributed/commands/table.c

```diff
@@ -1821,15 +1854,10 @@ GetSequenceOid(Oid relationId, AttrNumber attnum)
 	if (list_length(sequencesFromAttrDef) > 1)
 	{
 		/* to simplify and eliminate cases like "DEFAULT nextval('..') - nextval('..')" */
-		if (IsCitusTableType(relationId, CITUS_LOCAL_TABLE))
-		{
-			ereport(ERROR, (errmsg(
-								"More than one sequence in a column default"
-								" is not supported for adding local tables to metadata")));
-		}
 		ereport(ERROR, (errmsg(
 							"More than one sequence in a column default"
-							" is not supported for distribution")));
+							" is not supported for distribution "
+							"or for adding local tables to metadata")));
```

Comment:
Why did this change?

Reply:
Because I am now calling `GetSequenceOid` before checking the value of `includeSequenceDefaults` (for style reasons) in `pg_get_tableschemadef_string`, which is used in `GetShellTableDDLEventsForCitusLocalTable`, which is called before converting the table to type `CITUS_LOCAL_TABLE`. Hence we wouldn't reach the error for citus local tables.

> I am now calling `GetSequenceOid` before checking the value of `includeSequenceDefaults` (for style reasons)

I can revert this back if it's not a good error message and change the style in `pg_get_tableschemadef_string`